apps/bmd/src/pages/PollWorkerScreen.tsx

```diff
@@ -96,11 +107,76 @@ const PollWorkerScreen: React.FC<Props> = ({
 
   const currentDateTime = new Date().toLocaleString()
   const reportPurposes = ['Publicly Posted', 'Officially Filed']
+
+  const selectBallotStyle: EventTargetFunction = (event) => {
+    const { id = '' } = (event.target as HTMLElement).dataset
+    activateCardlessBallotStyleId(id)
+  }
+
+  if (ballotStyleId) {
+    return (
+      <React.Fragment>
+        <Screen>
+          <Main>
+            <MainChild center narrow>
+              <Prose>
+                <h1 aria-label="Ballot Style Activated">
+                  Ballot Style “{ballotStyleId}” Activated
+                </h1>
+                <ol>
+                  <li>Remove the poll worker card.</li>
+                  <li>
+                    Instruct the voter to press the{' '}
+                    <Text as="span" bold noWrap>
+                      “Start Voting”
+                    </Text>{' '}
+                    button.
+                  </li>
+                </ol>
+                <hr />
+                <p>
+                  To select another ballot style, you may{' '}
+                  <Button
+                    small
+                    onPress={() => {
+                      activateCardlessBallotStyleId('')
+                    }}
+                  >
+                    Deactivate Ballot Style “{ballotStyleId}”
+                  </Button>
+                </p>
+              </Prose>
+            </MainChild>
+          </Main>
+        </Screen>
+      </React.Fragment>
+    )
+  }
+
   return (
     <React.Fragment>
       <Screen flexDirection="row-reverse" voterMode={false}>
         <Main padded>
           <MainChild>
+            {isMarkAndPrintMode && isPollsOpen && (
+              <React.Fragment>
+                <Prose>
+                  <h1>Activate Ballot Style</h1>
+                </Prose>
+                <ButtonList data-testid="precincts">
+                  {precinctBallotStyles.map((bs) => (
+                    <Button
+                      data-id={bs.id}
+                      fullWidth
+                      key={bs.id}
+                      onPress={selectBallotStyle}
+                    >
+                      {bs.id}
+                    </Button>
+                  ))}
+                </ButtonList>
+              </React.Fragment>
+            )}
```

Comment:
Eventually, we will probably want to make a greater separation between these actions. Since at this point there are only two — and the open/close polls has a confirmation modal — I'm not concerned.